ibm_catalog.json

```diff
@@ -20,81 +20,55 @@
         "terraform",
         "solution"
       ],
-      "short_description": "Creates and configures a Secrets Manager private certificates engine.",
-      "long_description": "This deployable architecture is used to provision and configure a root certificate authority, an intermediate certificate authority and a certificate template. This Terraform-based automation is part of a broader suite of IBM-maintained Infrastructure as Code (IaC) asset collection, each following the naming pattern \"Cloud automation for *servicename*\" and focusing on single IBM Cloud service. These deployable architectures can be used on their own to streamline and automate service deployments through an [IaC approach](https://cloud.ibm.com/docs/secure-enterprise?topic=secure-enterprise-understanding-projects), or assembled together into a broader [automated IaC stack](https://cloud.ibm.com/docs/secure-enterprise?topic=secure-enterprise-config-stack) to automate the deployment of an end-to-end solution architecture.",
+      "short_description": "Creates and configures a Secrets Manager Private Certificates Engine",
+      "long_description": "This deployable architecture is used to provision and configure a root certificate authority, an intermediate certificate authority and a certificate template. \n\nℹ️ This Terraform-based automation is part of a broader suite of IBM-maintained Infrastructure as Code (IaC) assets, each following the naming pattern \"Cloud automation for *servicename*\" and focusing on single IBM Cloud service. These single-service deployable architectures can be used on their own to streamline and automate service deployments through an [IaC approach](https://cloud.ibm.com/docs/secure-enterprise?topic=secure-enterprise-understanding-projects), or assembled together into a broader [automated IaC stack](https://cloud.ibm.com/docs/secure-enterprise?topic=secure-enterprise-config-stack) to automate the deployment of an end-to-end solution architecture.",
```

Comment:
If possible, can we provide some more information? Description is not clarifying why it is a **Private** Certificate Engine. As of now, it appears like below and can be modified. 
<img width="439" height="185" alt="image" src="https://github.com/user-attachments/assets/8747b08a-4510-4b2a-a362-ea753f310abb" />